# Kallisto_Output_Handling
Modular pipeline to create kallisto output, rename default abundance.tsv and create master table for bulk input fastq's

# UNDER CONSTRUCTION....COMING SOON !

# Pipeline Objectives:

Kallisto, is most popular algorithm to quantify gene expression data for its fastness, accuracy and ease-of-use. It generates output in a folder specified by user with 3 files namely: (1) abundance.h5, (2) abundance.tsv, (3) run_info.json. One of the difficulties faced by user is that, there is no option to rename the default filenames generated by the program. Hence, it is challenging, when the user needs to run the analysis in batch mode for multiple datasets (for instance, 12 files at a time). 

You can find many discussions related to this topic in Kallisto forums as well as solutions with "Snakemake", which I am yet to learn. Thus, I am sticking to conventional coding method to solve this problem.

# Background:

Codes are written in Shell and Python 3xx




